sdk/eventhub/Azure.Messaging.EventHubs/samples/Program.cs

```diff
@@ -0,0 +1,229 @@
+                DisplayHelp();
+                return;
+            }
+
+            var samples = LocateSamples();
+
+            // Display the set of available samples and allow them to be run.
+
+            Console.WriteLine();
+            Console.WriteLine();
+            Console.WriteLine("=========================================");
+            Console.WriteLine("Welcome to the Event Hubs client library!");
+            Console.WriteLine("=========================================");
+            Console.WriteLine();
+            Console.WriteLine("Available Samples:");
+            Console.WriteLine();
+
+            for (var index = 0; index < samples.Count; ++index)
+            {
+                Console.WriteLine($"{ index + 1 }) { samples[index].Name }");
+                Console.WriteLine($"\t{ samples[index].Description }");
+                Console.WriteLine();
+            }
+
+            Console.WriteLine();
+            Console.Write("Please enter the number of a sample to run or press \"X\" to exit: ");
+
+            while(true)
+            {
+                var key = Console.ReadKey(true);
+
+                if ((key.KeyChar == 'x') || (key.KeyChar == 'X'))
+                {
+                    Console.Write(key.KeyChar);
+                    Console.WriteLine();
+                    Console.WriteLine();
+                    Console.WriteLine("Quitting...");
+                    Console.WriteLine();
+                    return;
+                }
+
+                if (Char.IsDigit(key.KeyChar))
+                {
+                    var choice = ((int)Char.GetNumericValue(key.KeyChar) - 1);
+
+                    if ((choice >= 0) && (choice < samples.Count))
+                    {
+                        Console.Write(key.KeyChar);
+                        Console.WriteLine();
+                        Console.WriteLine();
+                        Console.WriteLine();
+                        Console.WriteLine("-------------------------------------------------------------------------");
+                        Console.WriteLine($"Running: { samples[choice].Name }");
+                        Console.WriteLine("-------------------------------------------------------------------------");
+                        Console.WriteLine();
+
+                        await samples[choice].RunAsync(parsedArgs.ConnectionString, parsedArgs.EventHub);
+                        return;
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        ///   Displays the help text for running the samples to the console
+        ///   output.
+        /// </summary>
+        ///
+        private static void DisplayHelp()
+        {
+           Console.WriteLine();
+           Console.WriteLine($"{ typeof(Program).Namespace }");
+           Console.WriteLine();
+           Console.WriteLine("This executable allows for running of the Azure Event Hubs client library samples.  Because");
```

Comment:
```suggestion
           Console.WriteLine("This executable allows for running the Azure Event Hubs client library samples.  Because");
```

Reply:
Oh I like this suggested change feature I need to do that. :)